app/src/main/java/com/jc/android/plantdairy/data/remote/API.kt

```diff
@@ -0,0 +1,31 @@
+package com.jc.android.plantdairy.API
```

Comment:
Just give this file a more specific name, that would be great. :)

Reply:
Overall well done! Just a couple of minor suggestions then will be good to be merged.